### PR TITLE
Add region param for MinIO bucket

### PR DESCRIPTION
**Pull request by vviers**
As discussed in #359, we need to allow the setting of MinIO's `region` parameter

It's my first time actually touching the Grist codebase, so please do let me know if I need to add tests or to update docs :)

### CODE REVIEW

File: app/server/lib/configureMinIOExternalStorage.ts [hunk 1]
```diff
@@ -19,6 +19,11 @@ export function checkMinIOExternalStorage() {
     preferredEnvVar: 'GRIST_DOCS_MINIO_BUCKET',
   });
   if (!bucket) { return undefined; }
+  const region = settings.flag('bucket_region').requireString({
```
Comment (paulfitz) [inline app/server/lib/configureMinIOExternalStorage.ts:22]: Grepping for `settings.flag` it looks like it has been likeThisSoFar, so can you switch this to `bucketRegion`?

**Review by paulfitz — COMMENTED** [review]
Looks good, one consistency suggestion.

**Comment by paulfitz** [comment]
> It's my first time actually touching the Grist codebase, so please do let me know if I need to add tests or to update docs :)

The s3 integration is tested in `test/server/lib/HostedStorageManager.ts`, but it don't think it would be that easy to test region settings. I'm OK with not holding this PR up for that. If you have the energy to add to https://github.com/gristlabs/grist-help/blob/master/help/self-managed.md#how-do-i-set-up-snapshots--tag-core-tag-ee- that would be an excellent follow-up.

Thanks for doing this @vviers !

File: app/server/lib/configureMinIOExternalStorage.ts (hunk 1 above)
Comment (vviers) [reply app/server/lib/configureMinIOExternalStorage.ts]: Done ! Excuse my python ;)

**Review by paulfitz — APPROVED** [review]

**Pull request merged**